go.mod

```diff
@@ -38,6 +36,8 @@ require (
 	mvdan.cc/xurls/v2 v2.4.0
 )
 
+require github.com/golang-jwt/jwt v3.2.2+incompatible
```

Comment:
It looks this is an pretty old version of JWT. v4.0+ would have proper go module support. Maybe we can use that?

Reply:
Huh, no idea how this slipped through, thx!